tool/loadgen/main.go

```diff
@@ -18,6 +18,14 @@ const target = "http://app:8080"
 
 var hasRelay = os.Getenv("HAS_RELAY") == "true"
 
+type LoadGenOptions struct {
+	Path          string    `json:"path"`
+	ContainerName string    `json:"container_name"`
```

Comment:
Did you find this to be useful? It's going to be a string like `django-baseline-jio6rry`. With a quick check I think this string is not to be seen in the result, but appears in the (debug) compose file.

Reply:
I pulled it in because it was set from an env variable, but you're right that it doesn't provide much value. Will remove.